src/plcc/load_spec/validate_spec/validate_syntactic_spec/validate_rhs_test.py

```diff
@@ -23,28 +23,43 @@ def test_number_rhs_terminal():
     assert errors[0] == makeInvalidRhsTerminalFormatError(spec[0])
 
 def test_uppercase_rhs_alt_name():
+    word_setup = makeLine("<word> ::= ")
     invalid_alt_name = makeLine("<sentence> ::= <word>:Name")
-    Name1 = [
-        makeSyntacticRule(
-            invalid_alt_name,
-            makeLhsNonTerminal("sentence"),
-            [makeRhsNonTerminal("word", "Name")],
-        )
-    ]
+    Name1 = makeSyntacticSpec(
```

Comment:
I know I've waffled about becoming dependent on the parser... but we need to make these tests easier to read... plus, we can't validate much of anything if the parser doesn't work...

So, ideally each test should look like this...

```
def test_rhs_alt_names_cannot_start_with_uppercase():
    errors = validate('''
        <word> ::=
        <sentence> ::= <word>:Name
    ''')
    assert isinstance(errors[0], InvalidRhsAltNameFormatError)
```

Reply:
BTW, this is how I found the other bug.